src/redux/comicsReduser.ts

```diff
@@ -0,0 +1,51 @@
+import { ComicsSection } from "../ComicsPage";
+
+export const LOAD_COMICS = 'LOAD_COMICS';
+export const GET_COMICS_BY_ID = 'GET_COMICS_BY_ID';
+export const LOADING = 'LOADING';
+export const ERROR = 'ERROR';
+
+interface ComicsInitialState {
+  comics: Array<ComicsSection>,
+  loading: boolean,
+  error: Error | null
+}
+
+const initialState = {
+  comics: [],
+  loading: true,
+  error: null
+}
+
+export const comicsReducer = (state: ComicsInitialState = initialState, action: any) => {
+  switch (action.type) {
+    case LOAD_COMICS:
```

Comment:
name action types in this way please: GET_COMICS_REQUEST, GET_COMICS_SUCCESS, GET_COMICS_ERROR for better understanding